front/admin_ui/app.py

```diff
@@ -261,6 +315,51 @@ def query_jobs(pending_jobs_query):
             return {pending_jobs_query_result_df: gr.update(value=pd.DataFrame({"Error": [f"❌ {str(error)}"]}))}
         return {pending_jobs_query_result_df: gr.update(value=result)}
 
+    def refresh_dataset(
```

Comment:
I don't understand: is it an unintended copy/paste?

Reply:
Sorry, it was a mistake, I fixed it.